test/functional/action-params.spec.ts

```diff
@@ -72,6 +78,46 @@ describe("action parameters", () => {
         // reset metadata args storage
         defaultMetadataArgsStorage().reset();
 
+
+        @Middleware()
+        class SessionMiddleware implements MiddlewareInterface {
```

Comment:
This will cause some problems, `defaultMetadataArgsStorage().reset();` will clear all of the middleware.